include/evmc/evmc.h

```diff
@@ -160,10 +160,11 @@ typedef struct evmc_tx_context (*evmc_get_tx_context_fn)(struct evmc_context* co
  *  @param      context  The pointer to the Host execution context.
  *  @param      number   The block number. Must be a value between
  *                       (and including) 0 and 255.
+ *  @return              1 if exists, 0 otherwise.
```

Comment:
This comment is wrong. It should be something like `1 if the information is available, 0 otherwise` or something better.
Also:
- The comment for `number` is wrong. In this case leave "The block number" only and explain when the information is available in the main description.
- In the `result` explain that result is written only if the function returns 1 i.e. when information is available.